tracing/AsyncWrap/conceptual_overview.md

```diff
@@ -0,0 +1,73 @@
+## Concepts
+
+async\_hooks provides a mechanism to store context at the time of registration of a task (callback) and to restore context prior to execution of that task. This makes it possible to provide a consistent logical context across otherwise disjointed tasks.
```

Comment:
This is storing context something you can do with `async_hooks` but it is not what it directly provides. It provides full visibility of every asynchronous operation in node.js, what caused it, when it calls back, when the callback finished, and when it will no longer call back. 

A use case that "store context" does not cover is for example profiling, async\_hooks can be used to profile/estimate how much time each non-blocking action takes.